generators/java/templates/ext-app/README.md

```diff
@@ -0,0 +1,55 @@
+## Extension App Example
+This project is a simple complete example of a Symphony Extension App, comprising backend and frontend components.
+
+## Configuration
+If you have filled in the Generator with valid answers, the project is ready to go.
+To change these answers, please revise the configuration file located at [src/main/resources/application.yaml](src/main/resources/application.yaml).
+
+If you have changed the `appId`, please also change the values accordingly in the frontend components located in `src/main/resources/static/scripts`:
+- [app.js](src/main/resources/static/scripts/app.js)
+- [controller.js](src/main/resources/static/scripts/controller.js)
+
+Alternatively, run the generator again and revise your answers.
+
+## Getting Started
+### Using your company's pod
+1. Request your pod administrator to create an extension app in the Admin & Compliance Portal
+2. Supply them with an `appId` and a *public key*
+3. Ensure that the `appId` and *private key* in [application.yaml](src/main/resources/application.yaml) matches the pair above
+
+### Using the Developer Sandbox
+1. Use the shared `appId`: `localhost-10443`
+2. Download the [shared private key](https://localhost-10443.vercel.app/privatekey.pem) and use it to replace [rsa/privatekey.pem](rsa/privatekey.pem)
+
+## Run the Project
+### Using IDE:
+Load the project in your preferred IDE and launch the main class: `ExtensionAppApplication`
+### Using CLI:
+```bash
+$ ./mvnw spring-boot:run
+```
+
+## Run the Extension App
+1. Once the backend is up and running, visit https://localhost:10443/controller.html in a Chrome browser and bypass any security warnings.
```

Comment:
1. Once the backend is up and running, visit https://localhost:10443/controller.html in a Chrome browser and bypass any security warnings.
_This is only for development purposes and you will be expected to deploy the production application on a server with a trusted TLS certificate._
_Please note as well that using a bot is optional when creating an app. It is used only if you need a simple way to have a backend running together with your app and whether you want to use specific bot functionalities like a bot user identity, capable to send messages. However, if you don't have a bot created in the Admin and Compliance Portal that corresponds to the identity you provided on the [application.yaml](src/main/resources/application.yaml) file, you will need to remove the section of the bot that was automatically generated with the project in this application.yaml (bot/username and bot/privateKey/path)._